### PR TITLE
ngx.req.get_method can be used in log_by_lua*, marked it in docs

### PR DESCRIPTION
**Pull request by tokers**
I hereby granted the copyright of the changes in this pull request
to the authors of this lua-nginx-module project.

### CODE REVIEW

**Comment by agentzh** [comment]
@tokers `README.markdown` is automatically generated from `doc/HttpLuaModule.wiki`. Please edit the latter instead.

**Comment by tokers** [comment]
I am sorry, i will correct it.

**Comment by agentzh** [comment]
@tokers Thanks!

File: doc/HttpLuaModule.wiki [hunk 1]
```diff
@@ -3495,7 +3495,7 @@ This method does not work in HTTP/2 requests yet.
 == ngx.req.get_method ==
 '''syntax:''' ''method_name = ngx.req.get_method()''
 
-'''context:''' ''set_by_lua*, rewrite_by_lua*, access_by_lua*, content_by_lua*, header_filter_by_lua*, balancer_by_lua*''
+'''context:''' ''set_by_lua*, rewrite_by_lua*, access_by_lua*, content_by_lua*, header_filter_by_lua*, log_by_lua*, balancer_by_lua*''
```
Comment (thibaultcha) [inline doc/HttpLuaModule.wiki]: @tokers Will you please also regenerate the Markdown file via the [wiki2markdown.pl](https://github.com/openresty/openresty-devel-utils/blob/master/wiki2markdown.pl) script? Thanks.

Comment (tokers) [reply doc/HttpLuaModule.wiki]: @thibaultcha I have created another PR https://github.com/openresty/lua-nginx-module/pull/1395.

Comment (tokers) [reply doc/HttpLuaModule.wiki]: I will close this Pull Request.

**Pull request closed without merging**